app/src/DemoResetTask.php

```diff
@@ -1,71 +1,195 @@
 <?php
 
+use SilverStripe\Core\Path;
 use SilverStripe\Dev\BuildTask;
+use SilverStripe\PolyExecution\PolyOutput;
 use SilverStripe\ORM\DB;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Process\Process;
 
 class DemoResetTask extends BuildTask
 {
-    public function run($request)
-    {
-        $databaseConfig = DB::getConfig();
+    private static bool $can_run_in_cli = true;
 
-        $CLI_dbuser = escapeshellarg($databaseConfig['username']);
-        $CLI_dbpassArg = $databaseConfig['password'] ? escapeshellarg("-p" . $databaseConfig['password']) : '';
-        $CLI_database = escapeshellarg($databaseConfig['database']);
-        $CLI_server = escapeshellarg($databaseConfig['server']);
+    private static bool $can_run_in_browser = false;
```

Comment:
If this is meant to never ever be used in the browser, this should probably be a regular symfony command instead. The config is really intended to configure third-party tasks that get added from a module.
That said, this is fine since nobody else can see this code and it does what it needs to.

Reply:
Will leave as is